### ISSUE TITLE
Radar 15

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#284]
What are you working on this week and why?

[last week](https://github.com/gratipay/inside.gratipay.com/issues/283)

**Comment by whit537** [comment]
Map is the same afaict:

[![map](https://cloud.githubusercontent.com/assets/134455/8522270/4407185a-23ba-11e5-9ecf-be0c661ba479.png)](http://interactive.blockdiag.com/?compression=deflate&src=eJyFU01r3DAQvftXDC7k1AbTQMAsKaRuNwn00NYhBLo5aOXJRkQrufpYxwT_945sr9abmtYXe97MG715Y62l5s-VYBt4TcAi_w2_JFujvEhL5N4I16bvgWupzUX6LusfAhy-uAFsnoTDB6L6ekL1da2NSwl3yLYvMXFL0X1s2KKUunlIEtgJG2s-eysUWgt3wgqtYvXGIKrZo5-8O7CZZIpjBSWhlW7URH6eZ1lRzMr_eJrFFleGOVGzNoBHw5-dzbMT2Bjta3jdlxbFckmlizBXN5PN8-WSskF4N-r_8AnWloUXnToYMn6Th-7IQLhlz2iDuW8aZ1me07EJ9OMQPS2kJjM3K3XJufbK2WnyWlinTQsnK1X6teVG1I4MP6r5iTuBDfzw6HGKo-Onk9i5BDpSGkaImygvo3sGqznXcVvH8q_bWup2i8rBN9b8j1lzEZnfixv4Upb_psBfboUV9Usi0YvQcBH0dIPfNBMFwzn0LdSjpssxKKaY_g6jpaTaMRPF3FBMNydsR1S7A1zRYHSZVuoOjXgUnAWrQ9XY6jDNEPcN1E4LTvubdB-Rme3n-fk5XU5ayl5TEE4aev0jL2QHaD8C7a37A-PhSMQ)

**Comment by whit537** [comment]
We've got a PR for closing accounts that we need to land: https://github.com/gratipay/gratipay.com/pull/3601.

I'm hoping to get a PR for history pages today: https://github.com/gratipay/gratipay.com/issues/3455.

I've got four days this week so hopefully we can get some stuff done. Anyone going to be around to help me land some PRs? :sheep: :grin:

**Comment by kzisme** [comment]
I'll be around a lot - keeping up with everything.  

I'll probably look at the other extension issue at a minimum.  Anything else small worth noting?:hammer:

**Comment by whit537** [comment]
Inbox 5, GitHub 3, Support 80, [PRs](https://github.com/issues?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+user%3Agratipay+) 25, [Review](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+user%3Agratipay+label%3AReview) 5, Security 6-ish?, [Issues](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+user%3Agratipay+) 746

**Comment by whit537** [comment]
@kzisme Cool. Maybe https://github.com/gratipay/gratipay.com/pull/3565?

**Comment by whit537** [comment]
@kzisme I tried pinging you in Google Hangouts Chat. You around?

**Comment by kzisme** [comment]
@whit537 Yup just replied

**Comment by mattbk** [comment]
I'm keeping an eye on Freshdesk support tickets, but there are some on there that are beyond my ken (checking bank transfer records, scheduling PayPal payouts, etc.).  We can discuss at #280 if you'd prefer.  Other than that, probably not much time this week.

**Comment by whit537** [comment]
I'm going to keep working on https://github.com/gratipay/gratipay.com/pull/3618 for a bit this morning before heading over to meet @kaguillera at Blue Canary. I'll probably start self-merging PRs soon because I want to make sure we get some things out the door this week. I want to get account closing, history page updates (includes implementing double-entry accounting), and subscription page updates out the door before I get caught up on our support queue, since a lot of support queries depend on those.

When I'm able to digest support I'll review your work there, @mattbk, and see if there are ways we can make you more productive. Any time you're able to spend responding to people with a simple, "Sorry we're taking so long, we're trying to clean up some things, we'll get back to you again next week" message will help us out.

**Comment by mattbk** [comment]
That's pretty much what I've been doing. 

On July 15, 2015 8:17:11 AM CDT, Chad Whitacre <notifications@github.com> wrote:
>I'm going to keep working on
>https://github.com/gratipay/gratipay.com/pull/3618 for a bit this
>morning before heading over to meet @kaguillera at Blue Canary. I'll
>probably start self-merging PRs soon because I want to make sure we get
>some things out the door this week. I want to get account closing,
>history page updates (includes implementing double-entry accounting),
>and subscription page updates out the door before I get caught up on
>our support queue, since a lot of support queries depend on those.
>
>When I'm able to digest support I'll review your work there, @mattbk,
>and see if there are ways we can make you more productive. Any time
>you're able to spend responding to people with a simple, "Sorry we're
>taking so long, we're trying to clean up some things, we'll get back to
>you again next week" message will help us out.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/gratipay/inside.gratipay.com/issues/284#issuecomment-121611727

-- 
Sent from a phone that, although intelligent, is not street smart.

**Comment by whit537** [comment]
Awesome, thank you. :-)

!m @mattbk

**Comment by techtonik** [comment]
My queue:
* [ ] gratipay/algorithm.py#12 ready for review
* [ ] [export to PNG for blockdiag](https://bitbucket.org/blockdiag/blockdiag_interactive_shell/issues/4/save-as-png) - found a way to render SVG to canvas and then save as PNG - just need to code now. Learned some Clipboard API, and met with some SVG bugs on the way. Tried to use HTML imports from web components, but Firefox doesn't support them.

I am trying to get closer to client side javascript, and I'd like to visualize depersonalized data on GP 1.0 some day. I think that GP 2.0 is a necessary for us to survive, but it is not what people want. I still think we will resurrect GP one day.

http://www.forbes.com/sites/jurgenappelo/2015/07/08/the-peer-to-peer-bonus-system/
![image](https://cloud.githubusercontent.com/assets/515889/8708712/545a26f0-2b46-11e5-83aa-4210af73e00b.png)

http://www.forbes.com/sites/jurgenappelo/2015/07/08/the-peer-to-peer-bonus-system/

**Comment by whit537** [comment]
I'm back from the double-entry accounting :rabbit2: hole on #3618 to land a quick fix for history pages on #3616.

**Comment by kzisme** [comment]
I was considering opening a new issue, but I was curious if GSOC would be something we would want to do next year.  Last year when I looked applicants (the orginizations) they were getting prepared pretty early.  Would that be something you would be interested in @whit537

**Comment by rohitpaulk** [comment]
Just a word of caution - A GSoC program takes a lot of effort to manage, I doubt that we'll be able to pull it off in our current situation.

**Comment by whit537** [comment]
@kzisme Not a bad idea, but not something I'm ready to take on right now. I think @rohitpaulk is right and we should stabilize the project more first.

Okay! I've got https://github.com/gratipay/gratipay.com/pull/3616 ready to go. I'm gonna see about https://github.com/gratipay/gratipay.com/issues/3511 and then start merging PRs ...

**Comment by whit537** [comment]
I've self-merged [account closing](https://github.com/gratipay/gratipay.com/pull/3616) and [a tiny bugfix](https://github.com/gratipay/gratipay.com/pull/3611). I'm going to spend some time on [cached values](https://github.com/gratipay/gratipay.com/pull/3599), and then self-merge that (if I get far enough) along with [history cleanup](https://github.com/gratipay/gratipay.com/pull/3616). Then I'm going to break for lunch and relocate to a coffee shop, whence I'll [run payday](https://github.com/gratipay/gratipay.com/issues/3620) and then start thinking about next steps on [August 20](https://github.com/gratipay/gratipay.com/issues/3539), maybe getting into our support queue this evening. I guess I'd also like to rewrite the copy on the about pages and dust off stats as well. I think that could really help us not suck quite so bad and shouldn't be a ton of brain-work to get done before we start ramping up the communication engine ahead of August 20.

**Comment by whit537** [comment]
Breaking for lunch. Maybe I should keep going on cleanup today, tomorrow, and Monday. Next Tuesday is our target for communicating re: August 20.

**Comment by whit537** [comment]
Landed at this coffee shop. I'm releasing the PRs I merged earlier, then running [payday](https://github.com/gratipay/gratipay.com/issues/3620).

**Comment by mattbk** [comment]
Account closing works!  I plan on notifying everyone listed at https://github.com/gratipay/gratipay.com/issues/3461 and fully closing at least a few Freshdesk tickets today or tomorrow.

**Comment by whit537** [comment]
That sounds awesome, @mattbk. Thank you! :dancer:  

P.S. You might find one or two that want to close their account but own a team. Maybe reticket that from https://github.com/gratipay/gratipay.com/issues/3461?

**Comment by whit537** [comment]
Erm, we already have https://github.com/gratipay/gratipay.com/issues/3602. In that case maybe just copy over the notification list from https://github.com/gratipay/gratipay.com/issues/3461 as relevant.

**Comment by whit537** [comment]
Payday done.

**Comment by whit537** [comment]
Alright, where am I?

Inbox 5, GitHub 5, Support 19, Vendors 51, etc.

**Comment by whit537** [comment]
Alright, where am I?

Inbox 5, GitHub 5, Support 20, Vendors 53, etc.

**Comment by whit537** [comment]
I'm gonna zip through the Vendors queue. That should go pretty quick.

**Comment by mattbk** [comment]
Notifications done for https://github.com/gratipay/gratipay.com/issues/3461 related to account closing being back.

**Comment by whit537** [comment]
Yesssssssssssss.

!m @mattbk

**Comment by whit537** [comment]
https://www.stickermule.com/marketplace/1580-move-slow-and-make-things

**Comment by mattbk** [comment]
Notifications done for https://github.com/gratipay/gratipay.com/issues/3455 related to History pages being fixed.

**Comment by whit537** [comment]
@mattbk Killin' it! :sunglasses: 

I got https://github.com/gratipay/inside.gratipay.com/issues/249 over the hump and dealt with https://github.com/gratipay/inside.gratipay.com/issues/240#issuecomment-122333243.

**Comment by whit537** [comment]
The Freshdesk blog has good tips on running support. Their latest:

http://www.slideshare.net/Freshdesk/customer-servicetraining
http://blog.freshdesk.com/customer-service-training/

I've changed our subscription to their emails from vendors@ to support@ so the rest of yinz can benefit. :-)

**Comment by whit537** [comment]
Alright! We have 8 emails funneling into corresponding groups in Freshdesk:

 - support@
 - vendors@
 - sales@
 - safety@
 - security@

With @mattbk stepping up, I'm going to try paying attention to Freshdesk in two queues:

 - Support
 - Vendors, etc.

We're currently at Support 20, Vendors, etc. 15.

**Comment by whit537** [comment]
Weeded Vendors, etc. Down to 9.

**Comment by whit537** [comment]
Moved three tickets out of "Pending." I suppose we could start using that state, but we should document it on http://inside.gratipay.com/howto/support-users if we decide to do that. Support 23.

**Comment by mattbk** [comment]
Yeah, I'll start doing what step five says:
> Regardless of what action you take, “Close” the ticket when you're done
with it. If/when the user responds, it will be reopened automatically.

**Comment by whit537** [comment]
@mattbk Also not step 5 under "tips"

> Do *not* include the Freshdesk URL in tickets. It's not immediately useful to users, and is only likely to confuse them.

Freshdesk includes that automatically when you start a reply, so you have to remember to remove it. Skimming through your recent replies it looks like you haven't been doing that. Otherwise your replies are looking good so far, which is a big help, thanks. :-)

**Comment by whit537** [comment]
("note", not "not")

**Comment by whit537** [comment]
Worked on #255. Vendors, etc. 1.

**Comment by whit537** [comment]
It was mis-classified. Vendors, etc. 0! Support 28.

**Comment by whit537** [comment]
I've given @mattbk admin bit per https://github.com/gratipay/inside.gratipay.com/issues/280#issuecomment-122385332, and I've cycled out @seanlinsley @clone1018 and @Changaco:

```
=> select username from participants where is_admin;
┌─────────────────┐
│    username     │
├─────────────────┤
│ rohitpaulk      │
│ whit537         │
│ nplainsathletes │
└─────────────────┘
(3 rows)
```

**Comment by whit537** [comment]
![screen shot 2015-07-20 at 11 02 02 am](https://cloud.githubusercontent.com/assets/134455/8778967/c87d1c14-2ece-11e5-85f9-172682bfcc1b.png)
